src/modules/roc_core/target_posix/roc_core/colors.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019 Roc authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "roc_core/colors.h"
+
+namespace roc {
+namespace core {
+
+//! ANSI Color Codes.
+#define COLOR_NONE ""
+#define COLOR_START "\033["
+#define COLOR_END "m"
+#define COLOR_RESET COLOR_START "0" COLOR_END
+#define COLOR_SEPARATOR ";"
+#define COLOR_BOLD "1"
+#define COLOR_RED "31"
+#define COLOR_BLUE "34"
```

Comment:
Nitpick: could you please move these defines upper, between the include block and the namespace block? (according to our code-style; defines are global so it's a bit strange to have them in the namespace section).